src/main/java/org/apache/accumulo/examples/client/SequentialBatchWriter.java

```diff
@@ -53,28 +53,16 @@ public static Value createValue(long rowId) {
     private static final BatchWriterConfig BWDEFAULTS = new BatchWriterConfig();
 
     @Parameter(names = {"-t"}, required = true, description = "table to use")
-    public String tableName;
+    public String tableName = "batch";
 
-    @Parameter(names = {"--start"}, required = true, description = "starting line for the compaction")
-    public Integer start;
+    @Parameter(names = {"--start"}, required = true, description = "starting row")
+    public Integer start = 0;
 
-    @Parameter(names = {"--num"}, required = true, description = "number of lines")
-    public Integer num;
+    @Parameter(names = {"--num"}, required = true, description = "number of rows")
+    public Integer num = 10_000;
```

Comment:
Do we want to hardcode the number of rows or allow the user to use --num to define different rows as currently in the example?